huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go

```diff
@@ -527,7 +531,7 @@ func resourceGaussRedisInstanceV3Read(_ context.Context, d *schema.ResourceData,
 			d.Set("volume_size", volSize)
 		}
 		if specCode != "" {
-			logp.Printf("[DEBUG] Node SpecCode: %s", specCode)
+			log.Printf("[DEBUG] node specCode: %s", specCode)
 			d.Set("flavor", specCode)
```

Comment:
we should also use multierror.Append to get the errors.

Reply:
done